### PR TITLE
validate metadata.json as part of travis-ci testing

### PR DESCRIPTION
**Pull request by Phil-Friderici**
needed to let Travis validate the pettish metadata.json file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,7 +25,6 @@ doc/
 .DS_Store
 
 # Puppet
-metadata.json
```
Comment (Phil-Friderici) [inline .gitignore]: we need to track metadata.json now

**Comment by Phil-Friderici** [comment]
Spec test will be working after merging https://github.com/Ericsson/puppet-module-nisclient/pull/30

**Comment by jwennerberg** [comment]
Thanks. Can you please rebase this against master.

**Comment by Phil-Friderici** [comment]
rebased

File: metadata.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      "name": "pe",
+      "version_requirement": "3.x"
+    },
+    {
+      "name": "puppet",
+      "version_requirement": "3.x"
+    }
+  ],
+  "dependencies": [
+    {"name":"puppetlabs/stdlib","version_requirement":">= 3.2.0"},
```
Comment (ghoneycutt) [inline metadata.json]: versions should have an upper bound, suggest
stdlib @ `">= 3.2.0 < 6.0.0"`
rpcbind @ `">=1.1.0 < 2.0.0"`

Comment (Phil-Friderici) [inline metadata.json]: @ghoneycutt thanks for the hint, updated the PR

**Comment by Phil-Friderici** [comment]
@jwennerberg this patch is mostly included in https://github.com/Ericsson/puppet-module-nisclient/pull/32. Only the operatingsystem_support block in metadata.json is missing there.

**Comment by Phil-Friderici** [comment]
@jwennerberg Closing this PR, because I have added the missing part in https://github.com/Ericsson/puppet-module-nisclient/pull/33

**Pull request closed without merging**